programs/cipher/cipher_aead_demo.c

```diff
@@ -163,16 +167,18 @@ static int aead_prepare( const char *info,
  */
 static void aead_info( const mbedtls_cipher_context_t *ctx, size_t tag_len )
 {
-    // no convenient way to get the just cipher type (for example, AES)
-    const char *ciph = "???";
+    mbedtls_cipher_type_t type = mbedtls_cipher_get_type( ctx );
+    const mbedtls_cipher_info_t *info = mbedtls_cipher_info_from_type( type );
+    const char *ciph = mbedtls_cipher_info_get_name( info );
```

Comment:
Minor: this results in output like
```
AES-128-GCM, 128, GCM, 16
```
Either `AES, 128, GCM, 16` or `AES-128-GCM, 16` would make more sense.